packages/documentation/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "rules": {
+    "@typescript-eslint/no-unused-vars": "off"

```

Comment:
I guess this `no-unused-vars` is depreciated in the latest compiler !...can you cross check again

Reply:
the rule? the rule isn't deprecated, a bunch of discussion went on around it. there's a few reasons the eslint team keep it around instead of using the compiler options but we shouldn't use it anyway. ill do the change to use the compiler instead to detect these